cargo-concordium/src/main.rs

```diff
@@ -1200,6 +1220,22 @@ fn handle_run_v1(run_cmd: RunCommand, module: &[u8]) -> anyhow::Result<()> {
                              amount {} and parameter {:?}.",
                             address.index, address.subindex, name, amount, parameter
                         ),
+                        v1::Interrupt::Upgrade { module_ref } => eprintln!(
+                            "Contract was upgraded to module reference: {:?}.",
```

Comment:
Don't we have a display implementation? This will look somewhat ugly, will it not?

Reply:
I see we don't have the hex Display implementation.

Reply:
I just use hex::encode directly for now.